### ISSUE TITLE
`scoop update *` not  updating local bucket git repos

### ISSUE THREAD

**Issue by fluffynuts** [ScoopInstaller/Scoop#4936]
## Bug Report
I noticed today pwsh telling me there was an update (from my 7.2.3 to 7.2.4). So naturally, being aware of https://github.com/ScoopInstaller/Main/issues/3572, I crack open powershell.exe, `pskill pwsh` and `scoop update *` and _nothing happens_. Scoop claims it's all up to date.

So I deleted `~/.config/scoop/config.json` and try again - I get the message that the last update time isn't set and scoop proceeds to update _nothing_

So I had to head manually off into each of the bucket repos under `~/scoop` and `git pull --rebase`, after which `scoop update *` had _heaps_ of updates to do.

(I did also try from within pwsh 7.2.3 before getting brute-force on it, and again, no updates applied)

#### Current Behavior
Doesn't pull git changes to buckets, so updates don't happen.

#### Expected Behavior
Should pull the git changes & perform updates

#### Possible Solution
I dunno, this used to work.

### System details

**Windows version:** win11

**OS architecture:** 64bit

**PowerShell version:** 5.1.22000.653

**Additional software:** [(optional) e.g. ConEmu, Git]

#### Scoop Configuration
```json
{
    "SCOOP_BRANCH":  "master",
    "lastupdate":  "2022-05-18T09:46:04.1056781+02:00",
    "SCOOP_REPO":  "https://github.com/ScoopInstaller/Scoop"
}
```

**Comment by fluffynuts** [comment]
see also:
![image](https://user-images.githubusercontent.com/4669340/168987331-673515f7-d303-4c52-8fa4-f21f346432a7.png)

**Comment by fluffynuts** [comment]
I also can't just re-install:
![image](https://user-images.githubusercontent.com/4669340/168987597-69efec84-42cb-43b7-a163-35fa924375d0.png)

**Comment by fluffynuts** [comment]
I had to manually remove `~/scoop/shims/scoop*` and then run `Invoke-WebRequest get.scoop.sh | Invoke-Expression`; now `scoop update * works` 😕

**Comment by rashil2000** [comment]
Unable to reproduce on powershell.exe, Windows 11

**Comment by fluffynuts** [comment]
yeah, this is why I'm trying to transition to `winget` - "unable to repro, glhf"

this is far from the first issue I've raised and it's always @rashil2000 responding with "too bad"

**Comment by rashil2000** [comment]
> this is far from the first issue I've raised and it's always @rashil2000 responding with "sucks to be you"

Would you mind showing me one more instance where I did this? 

> no "gimme logs, I want to understand", 

What logs? Scoop does not log anything 

> no "do you have the scoop scripts that were failing", just "works on my machine". 

What scripts? I ran the all the exact commands you gave me, and showed you

> Not even a "can't repro, plz help".

That's precisely what I did. Do you want me to spell out "plz help"?

**Comment by fluffynuts** [comment]
> > this is far from the first issue I've raised and it's always @rashil2000 responding with "sucks to be you"
> 
> Would you mind showing me one more instance where I did this?
in the included link about pwsh: https://github.com/ScoopInstaller/Main/issues/3572 where I, and other users, have expressed that the current situation should be remedied: having a package manager which can install a package once but not update it is totally unexpected. Your responses there are basically "nope"
> 
> > no "gimme logs, I want to understand",
> 
> What logs? Scoop does not log anything
if your scoop works and mine does not, then one avenue I would take if I were you would be to ask for copies of my scoop scripts. Perhaps something got broken in an update?
> 
> > no "do you have the scoop scripts that were failing", just "works on my machine".
> 
> What scripts? I ran the all the exact commands you gave me, and showed you
the scripts that _are_ scoop. Since a hard reinstall fixed it, there must have been something wrong, no?
> 
> > Not even a "can't repro, plz help".
> 
> That's precisely what I did. Do you want me to spell out "plz help"?
incorrect: you literally just said "unable to reproduce on powershell.exe, Windows 11". Which anyone would tell you does _not_ include any attempt to gather more information or fix the problem. It just says "works for me":
![image](https://user-images.githubusercontent.com/4669340/169223184-b9dfbab2-de69-4e2a-8774-372bb9d5a6a2.png)
